### PR TITLE
Fix overly large image header

### PR DESCRIPTION
**Pull request by jpaddison3**

### CODE REVIEW

File: src/styles/partials/layout/_header.scss [hunk 1]
```diff
@@ -7,8 +7,7 @@
 }
 #header {
 	text-align: center;
-	min-height: 300px;
-	height: 70vh;
```
Comment (colophonemes) [inline src/styles/partials/layout/_header.scss]: Can we put the `vh`-specified height back. I'm happy with:

```sass
#header {
	...
	height: 50vh;
	@media(min-width: $screen-lg) {
		height: 40vh;
	}
}
```

File: src/layouts/layout/header.pug [hunk 2]
```diff
@@ -2,8 +2,9 @@ mixin headerLogo(type)
 	if breadcrumb_path.length && breadcrumb_path[0].file && breadcrumb_path[0].file.slug === 'eagx'
 		- type = 'eagx';
 	- type = type || 'eag'
+		//- TODO: There's no type for blog post list
 		#header-logo.row
-			.col-md-4.col-md-offset-4
+			.col-xs-8.col-xs-offset-2.col-sm-3.col-sm-offset-4
```
Comment (colophonemes) [inline src/layouts/layout/header.pug]: This makes the logo tiny at `sm` and `md` breakpoints. I'd revert this.

**Review by colophonemes — CHANGES_REQUESTED** [review]
Sorry, but big headers are dope and cool and I don't think you've thought sufficiently about how totally sweet they look. Props for taking the initiative on this though.

**Comment by jpaddison3** [comment]
Ready for another review @colophonemes

**Review by colophonemes — APPROVED** [review]
LGTM

**Pull request merged**